src/System.Drawing.Common/tests/SystemFontsTests.cs

```diff
@@ -37,22 +37,71 @@ public void SystemFont_Get_ReturnsExpected(Func<Font> getFont)
 
         public static IEnumerable<object[]> SystemFonts_WindowsNames_TestData()
         {
-            yield return Font(() => SystemFonts.CaptionFont, "CaptionFont", "Segoe UI");
-            yield return Font(() => SystemFonts.IconTitleFont, "IconTitleFont", "Segoe UI");
-            yield return Font(() => SystemFonts.MenuFont, "MenuFont", "Segoe UI");
-            yield return Font(() => SystemFonts.MessageBoxFont, "MessageBoxFont", "Segoe UI");
-            yield return Font(() => SystemFonts.SmallCaptionFont, "SmallCaptionFont", "Segoe UI");
-            yield return Font(() => SystemFonts.StatusFont, "StatusFont", "Segoe UI");
+            int userLangId = GetUserDefaultLCID();
+            SystemFontList fonts;
 
-            bool isArabic = (GetSystemDefaultLCID() & 0x3ff) == 0x0001;
-            yield return Font(() => SystemFonts.DefaultFont, "DefaultFont", isArabic ? "Tahoma" : "Microsoft Sans Serif");
+            switch (userLangId & 0x3ff)
+            {
+                case 0x11: //ja-JP (Japanese)
+                    fonts = new SystemFontList("Yu Gothic UI");
+                    break;
+                case 0x5C: //chr-Cher-US (Cherokee)
+                    fonts = new SystemFontList("Gadugi");
+                    break;
+                case 0x12: //ko-KR (Korean)
+                    fonts = new SystemFontList("맑은 고딕");
```

Comment:
We try to only use ASCII in the .cs files in this repo, as otherwise they often end up getting corrupted accidentally as files change encoding.  Can you use the Unicode values for these instead, e.g. `\xB9D1` instead of `맑`?

Reply:
Sure thing, just done it.

Reply:
Do you think it's better to leave `\x20` as a whitespace? I've escaped it as well because all other characters are escaped, but strictly speaking it is an ASCII character therefore can be left as is.

Reply:
`\x20` is fine